### DISCUSSION TITLE
New Dutch Translation for HFS 0.51.0

### DISCUSSION THREAD

**Discussion by PH5HP** [rejetto/hfs#419] [General]
Hi,

Updated translation for HFS 0.51.0

Kind regards,
Henk - fileserver.ph5hp.nl
[hfs-lang-nl 0-51-0.zip](https://github.com/rejetto/hfs/files/13787921/hfs-lang-nl.0-51-0.zip)

**Comment by rejetto** [comment]
thanks! included in next release
